src/GlobalExports.ts

```diff
@@ -27,3 +27,12 @@ export function lazyExport(component: IComponentDefinition, promiseResolve: Func
   Initialization.registerAutoCreateComponent(component);
   promiseResolve(component);
 }
+
+export function lazyExportModule(mod: any, promiseResolve: Function) {
```

Comment:
I hate the pattern of returning an empty promise.

The pattern of returning a promise is useful if you actually return something that can be modified down the line.